mix.exs

```diff
@@ -31,12 +31,20 @@ defmodule Appsignal.Plug.MixProject do
 
   # Run "mix help deps" to learn about dependencies.
   defp deps do
+    system_version = System.version()
+
+    mime_dependency =
+      case Version.compare(system_version, "1.10.0") do
+        :lt -> [{:mime, "~> 1.0"}]
```

Comment:
```suggestion
        :lt -> [{:mime, "~> 1.0", only: [:test, :test_no_nif]}]
```

Mime is a transient dependency through plug, not a direct dependency. It's a dependency for which we only need to specify a specific version on older Elixir versions in the test environment.

If we were to publish the original change, the `appsignal_phoenix` package would have an additional dependency for mime dependending on what version of Elixir it was published from.

This mix.exs file is not read dynamically upon install, but the compiled version is published to the hex.pm registry on publication.

Reply:
I understand. I removed the test environments because the dependency didn't come from a test dependency in these and the Phoenix one.

What you say totally makes sense. Thank you!